lib/kamal/cli/traefik.rb

```diff
@@ -11,20 +11,23 @@ def boot
 
   desc "reboot", "Reboot Traefik on servers (stop container, remove container, start new container)"
   option :rolling, type: :boolean, default: false, desc: "Reboot traefik on hosts in sequence, rather than in parallel"
+  option :confirmed, aliases: "-y", type: :boolean, default: false, desc: "Proceed without confirmation question"
   def reboot
-    mutating do
-      host_groups = options[:rolling] ? KAMAL.traefik_hosts : [ KAMAL.traefik_hosts ]
-      host_groups.each do |hosts|
-        host_list = Array(hosts).join(",")
-        run_hook "pre-traefik-reboot", hosts: host_list
-        on(hosts) do
-          execute *KAMAL.auditor.record("Rebooted traefik"), verbosity: :debug
-          execute *KAMAL.registry.login
-          execute *KAMAL.traefik.stop, raise_on_non_zero_exit: false
-          execute *KAMAL.traefik.remove_container
-          execute *KAMAL.traefik.run
+    confirming "Reboot Traefik on hosts? (stop container, remove container, start new container). This will cause a brief outage." do
```

Comment:
I was missing a question here (same as in other two places)

```suggestion
    confirming "Reboot Traefik on hosts? (stop container, remove container, start new container). This will cause a brief outage. Are you sure?" do
```

Reply:
@Sija I am not a native speaker, but there is already a question at the beginning of the phrase.

Reply:
We want something short, and finish with a question. 

@brunoprietog's point about rolling deploys is valid, though it doesn't guarantee that there won't be outages. Depending on how the load balancing is set up across hosts you could still get an errors for request in flight or while the load balancer spots the outage. The traefic reboot hooks are there so you can remove and drain the nodes before rebooting.

Maybe:
```
This will cause a brief outage on each host. Are you sure?
```

We can leave it to the user to infer whether those outages are concurrent or sequential based on the `rolling` argument.

Reply:
@djmb, you found the perfect words. I updated the text.